src/Zenaton/v1/Model/Scheduling/TaskTarget.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Zenaton\Model\Scheduling;
+
+use Zenaton\Services\Serializer;
+
+final class TaskTarget
+{
+    /** @var string */
+    private $name;
+    /** @var string */
+    private $programmingLanguage;
+    /** @var array */
+    private $properties;
+
+    private function __construct()
+    {
+    }
+
+    /**
+     * @internal should not be called by user code
+     *
+     * @return self
+     */
+    public static function fromArray(array $values = [])
```

Comment:
Could we do the same in the `__construct()` method instead? Or is there a case where we want to create a new instance without parameters?

Reply:
I removed all `::fromArray()` methods in favor of constructors :+1